components/core/src/clp_s/ColumnWriter.cpp

```diff
@@ -74,15 +83,14 @@ void ClpStringColumnWriter::store(ZstdCompressor& compressor) {
 }
 
 size_t VariableStringColumnWriter::add_value(ParsedMessage::variable_t& value) {
-    std::string string_var = std::get<std::string>(value);
-    uint64_t id;
-    m_var_dict->add_entry(string_var, id);
+    clp::variable_dictionary_id_t id{};
+    m_var_dict->add_entry(std::get<std::string>(value), id);
     m_variables.push_back(id);
-    return sizeof(int64_t);
+    return sizeof(clp::variable_dictionary_id_t);
 }
 
 void VariableStringColumnWriter::store(ZstdCompressor& compressor) {
-    size_t size = m_variables.size() * sizeof(int64_t);
+    size_t size = m_variables.size() * sizeof(clp::variable_dictionary_id_t);
```

Comment:
we can use auto?

it's not related to your change, but since we already have some auto used in this file, replacing size_t with auto doesn't harm